src/arrayMethodSort.js

```diff
@@ -5,7 +5,28 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    const defaultCompare = (firstElement, secondElement, compareFn) => {
+      if (compareFunction) {
+        return compareFn(firstElement, secondElement) > 0;
+      }
+
+      return String(firstElement) > String(secondElement);
+    };
```

Comment:
Would be better to declare it outside and use as default value for compareFunction.